components/keystone/aio-values.yaml

```diff
@@ -1085,3 +1076,28 @@ manifests:
   secret_registry: true
   service_ingress_api: true
   service_api: true
+
+# we don't want to enable OpenStack Helm's
```

Comment:
good description, consider rewrapping to more than 42 characters ;-)

Reply:
lol yeah I dunno why I did that. This is hopefully temporary. I'm discussing with upstream how to resolve this correctly.